### PR TITLE
systemd: version bumped to 229.

### PR DESCRIPTION
**Pull request by Florin65**

### CODE REVIEW

**Comment by lunar-linux-b01** [comment]
SUCCESS!

File: system/systemd/POST_INSTALL [hunk 1]
```diff
@@ -67,10 +67,10 @@ PRETTY_NAME="$DDESC"
 ANSI_COLOR="1;34"
 HOME_URL="http://lunar-linux.org/"
 EOF
-fi
+fi &&
+
+udevadm hwdb --update &&
 
-systemd-sysusers
```
Comment (Ratler) [inline system/systemd/POST_INSTALL]: Why was systemd-sysusers removed?

File: system/systemd/PRE_BUILD [hunk 2]
```diff
@@ -29,17 +29,23 @@ if mountpoint -q /usr && ! module_installed dracut; then
...
-patch_it $SOURCE2 1 &&
+default_pre_build
+
+# systemd guys wrongly assume that on every linux
+# installation is a group called "lock" this isn't
+# the case in lunar, so we switch it to root.
+sedit "s:0775 root lock:0775 root root:" tmpfiles.d/legacy.conf &&
+
+# Needed to migrate from udev module
+if module_installed udev; then
```
Comment (Ratler) [inline system/systemd/PRE_BUILD]: This is too early in the chain and "could" potentiella break a system with udev still on it. But maybe it's time that we just remove all udev / sysvinit related migration code from our modules. I doubt anyone is still running a lunar system without systemd in place.

File: system/systemd/DETAILS [hunk 3]
```diff
@@ -1,17 +1,14 @@
           MODULE=systemd
-         VERSION=228
+         VERSION=229
           SOURCE=$MODULE-$VERSION.tar.gz
-         SOURCE2=systemd-228-compat-1.patch
-         SOURCE3=systemd-228-sd-ndisc_networkd-link_fixes.patch
- SOURCE_URL_FULL=https://github.com/systemd/systemd/archive/v$VERSION.tar.gz
+         SOURCE2=systemd-225-compat-1.patch
```
Comment (Ratler) [inline system/systemd/DETAILS]: Out of curiosity why was the compat patch downgraded?

**Pull request closed without merging**